wagtail/admin/templates/wagtailadmin/workflows/task_chooser/includes/results.html

```diff
@@ -0,0 +1,62 @@
+{% load i18n wagtailadmin_tags %}
+{% if tasks %}
+    {% if searchform.is_searching %}
+        <h2 role="alert">
+        {% blocktrans count counter=tasks.paginator.count %}
+            There is {{ counter }} match
+        {% plural %}
+            There are {{ counter }} matches
+        {% endblocktrans %}
+        </h2>
+    {% else %}
+        <h2>{% trans "Tasks" %}</h2>
+    {% endif %}
+
+    <table class="listing">
+        <thead>
+            <tr class="table-headers">
+                <th>
+                    {% trans "Name" %}
+                </th>
+                {% if searchform.specific_task_model_selected %}
+                    {# TODO #}
```

Comment:
Going to leave this for a separate PR. We need to display some custom fields here and on the task index